aten/src/ATen/Context.cpp

```diff
@@ -117,6 +117,17 @@ void Context::setSDPUseMath(bool e) {
   enabled_mathSDP = e;
 }
 
+void Context::assignGlobalCtx(int pos, bool e){
```

Comment:
nit: I think some more description, and maybe longer argument names could be helpful for context

Reply:
Done